mod/state-transition/pkg/core/state_processor_payload.go

```diff
@@ -111,23 +115,10 @@ func (sp *StateProcessor[
 	_, _, _, _, _, _, _, _, _, _, _, _, _,
 ]) validateStatelessPayload(
 	blk BeaconBlockT,
-	nextPayloadTimestamp math.U64,
 ) error {
 	body := blk.GetBody()
 	payload := body.GetExecutionPayload()
 
-	// We skip timestamp check on Bartio for backward compatibility reasons
```

Comment:
moved to stateful validation because we need to pick up parent payload timestamp